src/esp/sensor/CameraSensor.cpp

```diff
@@ -107,6 +106,16 @@ gfx::RenderCamera* CameraSensor::getRenderCamera() const {
   return renderCamera_;
 }
 
+void CameraSensor::draw(scene::SceneGraph& sceneGraph,
+                        gfx::RenderCamera::Flags flags) {
+  for (auto& it : sceneGraph.getDrawableGroups()) {
+    // TODO: remove || true
```

Comment:
This TODO is getting carried around more and more places, are we ever going to actually do it?

Reply:
I believe the purpose is to build a filter function on drawables, which can be useful in the material editing.

Reply:
Yeah, the `it.second.prepareForDraw(*renderCamera_)` part of the if statement seems fine and I can see a use for that returning false and something not getting drawn, my concern is that we are simply ignoring the return value of that function via the `|| true`.  If we ever do want to do something with that function, we have a bunch of ` || true`'s that will make that harder.  

I don't know the history of the `|| true` (I think it's been there forever and thus likely a debugging thing that never got removed), but it currently does nothing since the only implementation for `prepareForDraw` just returns true.